app/javascript/listings/listings.jsx

```diff
@@ -284,27 +301,28 @@ export class Listings extends Component {
 
   listingSearch(query, tags, category, slug) {
     const t = this;
-    const { index, page, listings } = t.state;
-    const filterObject = { tagFilters: tags, hitsPerPage: 75, page };
-    if (category.length > 0) {
-      filterObject.filters = `category:${category}`;
-    }
-    index.search(query, filterObject).then(function searchDone(content) {
-      const fullListings = listings;
-      content.hits.forEach(listing => {
-        if (listing.bumped_at) {
-          if (!listings.map(l => l.id).includes(listing.id)) {
-            fullListings.push(listing);
-          }
-        }
-      });
+    const { page, listings } = t.state;
+    const dataHash = {
+      per_page: 75,
+      page,
+      classified_listing_search: query,
+    };
+
+    if (dataHash.tags) dataHash.tags = tags;
```

Comment:
The convention in the app so far seems to be this, so I'd prefer to keep it consistent. Again, I feel like we should have eslint enforce these kinds of things, but since it's not at the moment...🤷‍♀ 

```suggestion
    if (dataHash.tags) { dataHash.tags = tags; }
```

Reply:
Updated!